linux_os/guide/services/openstack/keystone/keystone_disable_user_account_days_inactive/rule.yml

```diff
@@ -17,7 +17,7 @@ rationale: |-
 severity: medium
 
 identifiers:
-    cce@osp12: 80570-5
```

Comment:
@shawndwells This change is interesting, could you please share with us what is the reason?

Reply:
That CCE was already used in RHEL content:

https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/services/sssd/sssd_enable_smartcards/rule.yml#L31